File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/prezentaOnline/DocumentPrezentaOnlineModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.prezentaOnline;

import java.util.Date;

public class DocumentPrezentaOnlineModel {

	private String comisieGl;
	private Long comisieGlId;
	private String numeDocument;
	private Date dataInceput;
	private Date dataSfarsit;
	private String documentLocationRealName;
	private String documentId;
	private boolean finalizata;

	public String getComisieGl() {
		return comisieGl;
	}

	public void setComisieGl(String comisieGl) {
		this.comisieGl = comisieGl;
	}

	public Long getComisieGlId() {
		return comisieGlId;
	}

	public void setComisieGlId(Long comisieGlId) {
		this.comisieGlId = comisieGlId;
	}

	public String getNumeDocument() {
		return numeDocument;
	}

	public void setNumeDocument(String numeDocument) {
		this.numeDocument = numeDocument;
	}

	public Date getDataInceput() {
		return dataInceput;
	}

	public void setDataInceput(Date dataInceput) {
		this.dataInceput = dataInceput;
	}

	public Date getDataSfarsit() {
		return dataSfarsit;
	}

	public void setDataSfarsit(Date dataSfarsit) {
		this.dataSfarsit = dataSfarsit;
	}

	public String getDocumentLocationRealName() {
		return documentLocationRealName;
	}

	public void setDocumentLocationRealName(String documentLocationRealName) {
		this.documentLocationRealName = documentLocationRealName;
	}

	public String getDocumentId() {
		return documentId;
	}

	public void setDocumentId(String documentId) {
		this.documentId = documentId;
	}

	public boolean isFinalizata() {
		return finalizata;
	}

	public void setFinalizata(boolean finalizata) {
		this.finalizata = finalizata;
	}
}

File: CloudFlow/FrontEnd-App/src/app/shared/model/prezenta-online/document-prezenta-online.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "@app/shared/json-mapper";
import { PrezentaSedinteCdPvgInvitatiARBReportFilterModel } from "../reports";
import { DateUtils } from "@app/shared/utils";

@JsonObject
export class DocumentPrezentaOnlineModel {

	@JsonProperty("comisieGl", String)
	public comisieGl: string = null;

	@JsonProperty("comisieGlId", Number)
	public comisieGlId: number = null;
	
	@JsonProperty("numeDocument", String)
	public numeDocument: string = null;

	@JsonProperty("dataInceput", JsonDateConverter)
	public dataInceput: Date = null;

	@JsonProperty("dataSfarsit", JsonDateConverter)
	public dataSfarsit: Date = null;

	@JsonProperty("documentLocationRealName", String)
	public documentLocationRealName: string = null;

	@JsonProperty("documentId", String)
	public documentId: string = null;

	@JsonProperty("finalizata", Boolean)
	public finalizata: boolean = false;

	public get dataInceputForDisplay(): string {
        return DateUtils.formatDateTimeForDisplay(this.dataInceput);
	}
	
	public get dataSfarsitForDisplay(): string {
        return DateUtils.formatDateTimeForDisplay(this.dataSfarsit);
	}
	
} 